### ISSUE TITLE
[doom] load order

### ISSUE THREAD

**Issue by oatmealm** [nobiot/md-roam#44]
Hi there!

How can I guarantee the load order in doom? Should I require md-roam in org-roam's after! Or use-package! call? I'm using the +roam flag and installing md-roam as per your instructions but  I can't create md files nor see existing one in the default org-roam-directory.

**Comment by nobiot** [comment]
I don't use +roam flag, and I defer org-roam -- I prefer to manually call M-x org-roam when I want to.
With this, I have the following in my config.el for Doom. By not having defer:, or command:, I can load md-load md-roam immediately.... 

If you have a need to load org-roam immediately, perhaps you can do (load-file) in custom: part of (use-package org-roam), perhaps? We (you and I) might need to dig deeper if that's the timing of your loading Org-roam.

(use-package! md-roam ; load immediately, before org-roam
  :config
  (setq md-roam-file-extension-single "md")
  (setq md-roam-use-org-extract-ref nil)
  (setq md-roam-use-org-file-links t)

;; The following Org-roam variables may interfere with Md-roam
  (setq org-roam-enable-fuzzy-links t)
  (setq org-roam-auto-replace-fuzzy-links nil))

**Comment by nobiot** [comment]
Additional question. Do you see "Md-roam is active" message in the echo area when you build Org-roam DB cache? 

If so... Md-roam is probably loaded correctly, but perhaps you would need to add some more variables in your config for Org-roam to recognise and extract markdown files...

**Comment by nobiot** [comment]
These: 
```
(setq md-roam-file-extension-single "md") 
  ;set your markdown extension
  ;you can omit this if md, which is the default.

(setq org-roam-title-sources '((mdtitle title mdheadline headline) (mdalias alias)))
  ;you need this as of commit `5f24103`.

(setq org-roam-file-extensions '("org" "md"))
```

**Comment by oatmealm** [comment]
> Additional question. Do you see "Md-roam is active" message in the echo area when you build Org-roam DB cache? 
> 

Thanks for the help!

Yes. I've continued tweaking and got it to work or at least it seems. Need to explain that I've set org-roam and md-roam to use a separate dn and folder where I want only md files. 

I do see the message when I build the db but nothing else seems to work. Trying to create a note fails with and error about the capture string (which is the same of course and works in org-roam. Also side bar is empty (no header) when I open an md file manually or using org-roam-find-file. 

I'll upload my current config using .dir-locals a little bit later when I'm home. 

Thanks!
